app/Console/Commands/Telegram/StatsCommand.php

```diff
@@ -75,9 +77,12 @@ public function handle($arguments)
 
 		//Get the nb of users
 		preg_match('/class="e-f-ih" title="(\d*\,\d+)/', $file_string, $users);
+		preg_match('/class="daily-users" title="(\d*\,\d+)/', $file_string_f, $usersf);
 		$users = $users[1];
+		$usersf = $usersf[1];
 
-		$this->replyWithMessage("Existem $users utilizadores com um rating médio de $rating");
+		$this->replyWithMessage("Existem $users[1] utilizadores no Chrome e $usersf com um rating médio no Chrome de $rating");
```

Comment:
You're defining `$users` to be `$users['1']`, so `$users` is no longer an array, and this will return an error. Also, you're missing the `Firefox` word on the string. It should be `.. $usersf no Firefox, com um...`.